### ISSUE TITLE
Suggestion (Bot) - Member List Suggestions.

### ISSUE THREAD

**Issue by gxdsetmxnsters** [OctoconDev/issues#363]
This is moreso a handful of suggestions concerning the member list layout. If I need to make individual suggestions do let me know.

1. Perhaps adding better pagination for people who are larger systems ('last' button that immediately goes to the last page or a way to search for a specific page to be able to go to), as of currently paging through multiple pages of the member list to look for one members ID or to check who does and doesn't have a proxy is extremely tedious.

2. The ability to hide pronouns from the member list. It's fairly difficult to parse out the amount of info on the member list and feels rather cluttered, so being able to either disable pronouns from showing up on the list and / or removing them entirely would be nice.

3. Instead of having the ID/alias show up, have it be the alias unless there's none present, in which it shows the member ID (on the member list).

**Comment by atlas-oc** [comment]
`/alter list` was fully redesigned a few weeks ago, taking many of these suggestions into account!
